htdocs/admin/boxes.php

```diff
@@ -112,7 +112,7 @@
                         }
                         else dol_print_error($db);
 
-                        $sql = "INSERT INTO ".MAIN_DB_PREFIX."boxes (";
+                        $sql = "INSERT IGNORE INTO ".MAIN_DB_PREFIX."boxes (";
```

Comment:
Does this work with pgsql ? Is this reserved word a sql standard reserved word or specific to mysql syntax ?

Reply:
after some research, it's not working on pgsql.

The other solution is to verify that the record does not exist before insert.

Reply:
@eldy B plan implemented ;)